### DISCUSSION TITLE
Batch mode cannot be killed with ctrl+c

### DISCUSSION THREAD

**Discussion by dan1elhughes** [jkwill87/mnamer#124] [Questions]
👋 I started a batch mode run and wanted to cancel it. Neither Ctrl+D nor Ctrl+C would let me kill it.

**Comment by jkwill87** [comment]
mnamer doesn't have any logic which should interfere with interrupts when running in batch mode. Without blocking for user input (as is done for interactive mode) I don't think that there is anything that be done here.

AFAIK its a platform or interpreter issue rather than an mnamer one. For instance, interrupting `while True: pass`, the more contrived Python loop I can muster, seems to work as intended on MacOS and Linux but not on Windows 🤷‍♂️. If you're running a UNIX-based platform then CTRL-Z `kill %1` may work instead, however.
